src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -478,6 +485,39 @@ private InitCodeLineView generateMapInitCodeLine(MethodContext context, InitCode
     return surfaceLine.build();
   }
 
+  // TODO(hzyi): generate necessary imports
+  private InitCodeLineView generateReadFileInitCodeLine(MethodContext context, InitCodeNode item) {
+    ReadFileInitCodeLineView.Builder surfaceLine = ReadFileInitCodeLineView.newBuilder();
+    SurfaceNamer namer = context.getNamer();
+    ImportTypeTable typeTable = context.getTypeTable();
+    checkState(
+        item.getType().isBytesType(),
+        "Error setting init code node %s to reading from file. "
```

Comment:
A user may not node about "init code node", so I suggest rephrasing slightly:
"Error setting \"%s\" to be read from file."